MediaPlayer/MediaPlayer.js

```diff
@@ -77,6 +82,14 @@ const MediaPlayerBase = kind({
 		 */
 		currentTime: PropTypes.number,
 
+		/**
+		 * Removes interactive capability from this component.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		disabled: PropTypes.bool,
```

Comment:
although this prop is added in Mediaplayer, The MediaSlider and MediaControls can still be changed via click/ 5-way. I think you need to pass this props to MediaSlider and all Buttons in MediaControls